pkg/metrics/metrics.go

```diff
@@ -58,29 +58,29 @@ var (
 
 func init() {
 	M_Instance_Start = prometheus.NewCounter(prometheus.CounterOpts{
-		Name: "instance_start_total",
+		Name: "grafana_instance_start_total",
```

Comment:
Typically we use the Prometheus `Namespace` in the [Opts](https://godoc.org/github.com/prometheus/client_golang/prometheus#Opts) struct.

Here's another example: 
https://github.com/prometheus/mysqld_exporter/blob/master/collector/exporter.go#L78-L83